db/schema.rb

```diff
@@ -338,6 +337,7 @@
     t.integer  "parent_id"
     t.boolean  "quantity_is_decimal", default: false
     t.string   "unique_code"
+    t.integer  "position"
```

Comment:
This new column is meant to be used for displaying **fees/fee types** without relying on the order they've been inserted in the DB (which was the case until now). There's a potential for this to be indexed, but would like to understand a bit better what other columns are used as well for filtering.